projects/plugins/boost/app/modules/optimizations/minify/Concatenate_JS.php

```diff
@@ -248,12 +248,10 @@ public function do_items( $handles = false, $group = false ) {
 				if ( isset( $href ) ) {
 					$handles = implode( ',', $js_array['handles'] );
 
-					$load_mode = jetpack_boost_page_optimize_load_mode_js();
-
 					if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {
-						$tag = "<script data-handles='" . esc_attr( $handles ) . "' $load_mode type='text/javascript' src='$href'></script>\n";
+						$tag = "<script data-handles='" . esc_attr( $handles ) . "' type='text/javascript' src='$href'></script>\n";
 					} else {
-						$tag = "<script type='text/javascript' $load_mode src='$href'></script>\n";
+						$tag = "<script type='text/javascript' src='$href'></script>\n";
```

Comment:
`$href` is missing escaping

Reply:
Fixed with 7b43459a3c8fd7c5bc4d0c9cb2da98ed1c4c2808.